docs/man/docker.1.md

```diff
@@ -121,15 +127,12 @@ unix://[/path/to/socket] to use.
   Use TLS and verify the remote (daemon: verify client, client: verify daemon).
   Default is false.
 
+**--userland-proxy**=*true*|*false*
+    Rely on a userland proxy implementation for inter-container and outside-to-container loopback communications. Default is true.
+
```

Comment:
I'd simplify the first part, e.g.

"Enable or disable the userland-proxy for inter-container and outside-to-container loopback communication. Default is true (enabled)."

Not sure about the "outside-to-container loopback communication" part, thinking of better wording for that :)